.github/workflows/desktop-build.yml

```diff
@@ -251,8 +251,8 @@ jobs:
           install_qt: ${{matrix.qt_version}}
         run: |
           brew update
-          brew install protobuf@21
-          brew link --force protobuf@21
+          brew install protobuf
+          brew link --force protobuf
```

Comment:
This does break macOS 10.15 now.

Homebrew uses protobuf 24 which is not available for anything below macOS 12.